build/updatehistorictzdb.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+rm -rf tmp
+mkdir tmp
+
+# Get the list of files, just once.
+
+mkdir -p tmp/releases
+cd tmp/releases
+FILES=files.txt
```

Comment:
Prefer to mark variables as readonly where appropriate.

If you're happy with bashisms, then

```bash
declare -r FILES=files.txt
```

otherwise

```sh
FILES=files.txt
readonly FILES
```

(likewise other constant-like variables below.)